src/main/java/org/desarrolladorslp/workshops/springboot/forms/ColumnForm.java

```diff
@@ -0,0 +1,22 @@
+package org.desarrolladorslp.workshops.springboot.forms;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.desarrolladorslp.workshops.springboot.validation.ValidationCreate;
+import org.desarrolladorslp.workshops.springboot.validation.ValidationUpdate;
+
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+
+@Data
```

Comment:
Nice... I think you'll need to include some slides to talk about Lombok (in case you hadn't considered it yet) and some time to install IDE plugins